IBPSA/Utilities/IO/SignalExchange/WeatherStation.mo

```diff
@@ -0,0 +1,316 @@
+  connect(weaBus.cloTim, reaWeaCloTim.u) annotation (Line(
+      points={{-100,0},{-60,0},{-60,76},{30,76},{30,90},{38,90}},
+      color={255,204,51},
+      thickness=0.5), Text(
+      string="%first",
+      index=-1,
+      extent={{-6,3},{-6,3}},
+      horizontalAlignment=TextAlignment.Right));
+  connect(weaBus.lat, reaWeaLat.u) annotation (Line(
+      points={{-100,0},{-60,0},{-60,46},{30,46},{30,60},{38,60}},
+      color={255,204,51},
+      thickness=0.5), Text(
+      string="%first",
+      index=-1,
+      extent={{-6,3},{-6,3}},
+      horizontalAlignment=TextAlignment.Right));
+  connect(weaBus.lon, reaWeaLon.u) annotation (Line(
+      points={{-100,0},{-60,0},{-60,16},{30,16},{30,30},{38,30}},
+      color={255,204,51},
+      thickness=0.5), Text(
+      string="%first",
+      index=-1,
+      extent={{-6,3},{-6,3}},
+      horizontalAlignment=TextAlignment.Right));
+  connect(weaBus.nOpa, reaWeaNOpa.u) annotation (Line(
+      points={{-100,0},{-60,0},{-60,-14},{30,-14},{30,0},{38,0}},
+      color={255,204,51},
+      thickness=0.5), Text(
+      string="%first",
+      index=-1,
+      extent={{-6,3},{-6,3}},
+      horizontalAlignment=TextAlignment.Right));
+  connect(weaBus.solAlt, reaWeaSolAlt.u) annotation (Line(
+      points={{-100,0},{-60,0},{-60,-44},{30,-44},{30,-30},{38,-30}},
+      color={255,204,51},
+      thickness=0.5), Text(
+      string="%first",
+      index=-1,
+      extent={{-6,3},{-6,3}},
+      horizontalAlignment=TextAlignment.Right));
+  connect(weaBus.solDec, reaWeaSolDec.u) annotation (Line(
+      points={{-100,0},{-60,0},{-60,-74},{30,-74},{30,-60},{38,-60}},
+      color={255,204,51},
+      thickness=0.5), Text(
+      string="%first",
+      index=-1,
+      extent={{-6,3},{-6,3}},
+      horizontalAlignment=TextAlignment.Right));
+  connect(weaBus.solHouAng, reaWeaSolHouAng.u) annotation (Line(
+      points={{-100,0},{-60,0},{-60,-100},{30,-100},{30,-90},{38,-90}},
+      color={255,204,51},
+      thickness=0.5), Text(
+      string="%first",
+      index=-1,
+      extent={{-6,3},{-6,3}},
+      horizontalAlignment=TextAlignment.Right));
+  connect(weaBus.solTim, reaWeaSolTim.u) annotation (Line(
+      points={{-100,0},{-60,0},{-60,76},{70,76},{70,90},{78,90}},
+      color={255,204,51},
+      thickness=0.5), Text(
+      string="%first",
+      index=-1,
+      extent={{-6,3},{-6,3}},
+      horizontalAlignment=TextAlignment.Right));
+  connect(weaBus.solZen, reaWeaSolZen.u) annotation (Line(
+      points={{-100,0},{-60,0},{-60,46},{70,46},{70,60},{78,60}},
+      color={255,204,51},
+      thickness=0.5), Text(
+      string="%first",
+      index=-1,
+      extent={{-6,3},{-6,3}},
+      horizontalAlignment=TextAlignment.Right));
+  annotation (Documentation(info="<html>
+<p>
+This block enables the reading of weather measurements and their meta-data by an external
+program without the need to explicitly propogate the signal to a top-level model.
```

Comment:
Shouldn't this be "propagate"? We have this typo in various models.

Reply:
Fixed in https://github.com/ibpsa/modelica-ibpsa/pull/1403/commits/97e362e1c1789e7ccf2baff623a9715ebe0b04d7.